src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/LanguageServices/ProjectContextCodeModelProvider.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.VisualStudio.ProjectSystem.VS.LanguageServices
     /// </summary>
     [Export(typeof(ICodeModelProvider))]
     [Export(typeof(IProjectCodeModelProvider))]
-    [AppliesTo(ProjectCapability.DotNetLanguageService)]
+    [AppliesTo(ProjectCapability.CSharpAndVisualBasic)]
```

Comment:
This should be CSharp _or_ Visual Basic, right? I don't think a single project will ever have both

Reply:
Also, perhaps `CSharpOrVisualBasicLanguageService` would be a good fit, since previously `LanguageService` was checked. Unless that is part of the problem?

Reply:
@davkean 
your comment in here says :  turned on for C#  and  VB
https://github.com/dotnet/project-system/issues/3664#issuecomment-579005448

Do you wanted to say OR ?

Reply:
This must include LanguageService, to avoid being called for shared projects. Shared Projects get renamed by the projects that consume them.

Reply:
Sorry, I misread the comment/change - I thought it was rename.

@davidwengier is right it must be CSharpOrVisualBasicLanguageService, there's never a case that a project is both C# and VB at the same time.